releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -3,7 +3,15 @@ Azure App Configuration Push extension for Azure DevOps pipeline can be installe
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
 ### v5.0.0 - February, 02 2023
-Updated the task to require Node.js 16. It previously required 10
+* Updated the task to require Node.js 16. It previously required 10

```

Comment:
This is a breaking change too.

Reply:
Yes, updated to document the breaking change.